src/main/java/org/candlepin/subscriptions/subscription/SubscriptionServiceProperties.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.subscription;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * Additional properties related to the Subscription Service
+ */
+@Getter
+@Setter
+@ToString
+public class SubscriptionServiceProperties {
+
+    /**
+     * Number of times we should try requesting info from the Subscription Service if something fails.
+     */
+    private int maxAttempts = 4;
+
+    /**
+     * The initial sleep interval between retries when retrying fetching info from the Subscription Service
+     */
+    private long backOffInitialInterval = 1000L;
```

Comment:
Can you please define these two in the application.yaml? Also, I prefer using `Duration` for the backOffInitialInterval.